.travis.yml

```diff
@@ -1,6 +1,24 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-script: bash -ex .travis-opam.sh
-sudo: true
+sudo: false
+services:
+    - docker
+install:
+    - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
+script:
+    - bash -ex .travis-docker.sh
 env:
-        - OCAML_VERSION=4.02 PACKAGE=xapi-rrdd-plugin EXTRA_REMOTES=git://github.com/xapi-project/opam-repo-dev
+    global:
+        - OCAML_VERSION=4.04.2
+        - DISTRO=debian-stable
```

Comment:
just a question, why do we need to designate a distro here, and it is debian?

Reply:
It is required by the ocaml travis-docker script. We could put anything really. Depending on the package we have centos, debian-stable and debian-unstable. I agree that at a certain point we should settle on a specific one and use that one everywhere

Reply:
Debian stable is good for stability, because it will automatically track the current stable version of Debian, so it won't get outdated. On the other hand, if we use CentOS, that's closer to XenServer. But we already use CentOS for the Travis build of xs-opam.

Reply:
When we use the travis-docker script, the Travis build is Docker-based, and therefore we can select which base distribution we want for the container.

Reply:
I think this is the list of images one can use: https://github.com/ocaml/opam-dockerfiles